site/content/docs/5.3/examples/form/form.js

```diff
@@ -16,6 +16,16 @@
         // Focus on first error
         const invalidItems = form.querySelectorAll(':invalid')
         invalidItems[0].focus()
+        // Add the id of the corresponding invalid message to each invalid field
+        invalidItems.forEach(element => {
+          const valuesArray = [`${element.id}Label`, `${element.id}Feedback`].join(' ')
```

Comment:
What if folks have several feedback for one input ?
Not a big fan of `${element.id}Label` and so on since it's not flexible at all + doesn't work for several feedback as well.